sdk/test-utils/recorder-v2/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "extends": "../../../tsconfig.package",
+  "compilerOptions": {
+    "declarationDir": "./types",
+    "target": "es5",
+    "outDir": "./dist-esm",
+    "lib": [
+      "dom",
+      "es5",
+      "es6",
+      "es7",
+      "esnext"
```

Comment:
Should add ES2017 to this list and not everything else. We don't want to add lib entries for things that aren't actually supported in our runtime. FWIW these are also self-referential. Including "ESNext" includes everything under it. So this should just be `"lib": ["dom", "es2017"]` .